game/models.py

```diff
@@ -585,13 +586,44 @@ class Meta:
         )
 
 
-class CompetitionManager(models.Manager):
+class CompetitionMixin(object):
```

Comment:
this https://docs.djangoproject.com/en/1.7/topics/db/managers/#modifying-initial-manager-querysets